activesupport/lib/active_support/multibyte/chars.rb

```diff
@@ -5,6 +5,11 @@
 require "active_support/core_ext/string/behavior"
 require "active_support/core_ext/module/delegation"
 
+ActiveSupport.deprecator.warn(
+  "ActiveSupport::Multibyte::Chars and String#mb_chars are deprecated and will be removed in Rails 8.2. " \
```

Comment:
@byroot I had an issue with a gem ( `carrierwave` ) https://github.com/carrierwaveuploader/carrierwave/pull/2773/

and realized maybe you also want to deprecate the `core_ext` as well  here : https://github.com/rails/rails/blob/main/activesupport/lib/active_support/core_ext/string/multibyte.rb

Reply:
We could, but I thought it would be too noisy? The first call to `mb_chars` will trigger loading of `Multibyte` and emit the deprecation, I feel it's enough?

Reply:
Yes you're right